src/content/en/agencies/partials/web-directory.html

```diff
@@ -0,0 +1,75 @@
+{% framebox class="directory-framebox" height="auto" enable_widgets="true" %}
+  <head>
```

Comment:
frameboxes do not need the `<head>` or `<body>` tags, everything is placed in the body automatically.